test/screenplay_web/plugs/ensure_api_auth_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule ScreenplayWeb.EnsureApiAuthTest do
+  use ScreenplayWeb.ConnCase
+
+  describe "init/1" do
+    test "passes options through unchanged" do
+      assert ScreenplayWeb.EnsureScreenplayEmergencyAdminGroup.init([]) == []
```

Comment:
Maybe not the module you meant to test 😛

Reply:
🤦